src/components/widgets/WinrateChart.tsx

```diff
@@ -63,6 +64,8 @@ export function WinrateChart(props: WinrateChartProps) {
       },
       x: {
         display: true,
+        type: "time",
+        time: { unit: "day", tooltipFormat: "d MMM, y" },
         ticks: {
```

Comment:
Can we revert this? We can add Timescale support to this chart type when we are sure we need it.

I think we may want to feature these on the homepage at some point and I rather have the backend formatted data where we can.